desk/lib/bolt.hoon

```diff
@@ -63,35 +60,35 @@
     ?>  =(src.bowl our.bowl)
     =/  =bean  !<(bean vase) 
     ?-  -.bean
-        %toggle-which
-      =.  which  switch.bean
-      :-  ~[(emit which/b+which)]
-          this 
+        %toggle-white
+      =.  white  switch.bean
+      :_  this
+      ~[(emit:json white/b+white)]
     ::
         %toggle-kids
       =.  kids  switch.bean
-      :-  ~[(emit kids/b+kids)]
-          this
+      :_  this
+      ~[(emit:json kids/b+kids)]
     ::
         %add-white
-        =.  users.whitelist  (~(uni in users.whitelist) users.bean)
-        :_  this           
-            ~[(emit add-users-white/(ships:util:json users.whitelist))]
+      =.  whitelist  (~(uni in whitelist) users.bean)
+      :_  this           
```

Comment:
Given that you indented all the other cases properly I assume this was just a mistake, but: in `?-` and `?+`, you have two choices:

```hoon
?-  value
  %case0  result0
  %case1  result1
==
```
or
```hoon
?-    value
    %case0
  result0
:: Comment line optional but recommended. Align it with the wut rune!
    %case1
  result1
==
```

These can be combined, BUT for each case, you want to keep this pattern. Move back one step.